src/veh_interact.cpp

```diff
@@ -18,6 +18,8 @@
 #define ISNAN std::isnan
 #endif
 
+static const std::set< ammotype > unsiphonable = { "gasoline", "diesel", "battery", "plut_cell", "plasma" };
```

Comment:
Yeah, undrainable.

And there's also one (small) difference more: siphon can be used to target another vehicle.

I'll merge them, then.